mautic/mautic
# Allow files in a form submission to be send as attachments

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes 
| Automated tests included? | No
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | No
| Deprecations? | No

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
We use forms heavily for all of our customers. Sometimes users can upload attachments, for example a CV when applying for a job. These form submissions are then sent to the HR, but these people don't necessary have access to Mautic so they can't download or read the CV.

This PR provides a simple solution: You can configure Mautic to instead attach all files to the email, so it can be easy sent/forwarded to anybody.

![Screenshot 2019-07-24 at 13 28 09](https://user-images.githubusercontent.com/1575445/61790207-ecfc9300-ae16-11e9-8684-ea177603dcad.png)


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create a form with a file field
3. Add an action "Send form result" and enable the checkbox "Attach file in email"
4. Go to preview and submit the form
5. Check your emails  

Disclaimer: sponsored by [maatoo.io](https://maatoo.io)
